docs/source/Migrations.md

```diff
@@ -1,17 +1,137 @@
+      revision   Create a blank migration file.
+      status     Show information about migration status.
+      upgrade    Run database upgrades.
+
+
+
 ## Create Migrations
 
-To create migrations for your model changes, use:
+To create the initial migrations for models, use:
 
-    ./${PACKAGE}/manage.py makemigrations
+    ./${PACKAGE}/manage.py db create .models
 
-You can optionally specify a name for your migrations:
+Where `.models` represents the python models module.
 
-    ./${PACKAGE}/manage.py makemigrations somename
 
 ## Apply Migrations
 
 To apply migrations on the currently configured database, use:
 
-    ./${PACKAGE}/manage.py migrate
+    ./${PACKAGE}/manage.py db upgrade
+    
+
+## Upgrade model
+
+Peewee Moves support only creating a new model. The alteration of an existing model is done manually
+by generating a new revision and editing it. For generating an empty migration use:
+
+    ./${PACKAGE}/manage.py db revisions alter_model
+    
+Then update the newly generated file `migrations/0XXX_alter_model.py` to contains the model changes.
+The documentation of `peewee-moves` Migrator API can be found
+[here](https://peewee-moves.readthedocs.io/en/latest/usage.html#migrator-api).
+
+
+
+### Adding/Removing/Renaming a model field
+
+Create an empty revision file with the above command `migrations/0004_alter_user_fields.py` then
+edit the `upgrade` and `downgrade` sections as follows:
+
+```python
+def upgrade(migrator):
+    # Add display_name field
+    migrator.add_column('user', 'display_name', 'char', max_length=58, null=True)
+    
+    # Remove role field
+    migrator.drop_column('user', 'role')
+    
+    # Rename key field to password
+    migrator.rename_column('user', 'key', 'password')
+
+def downgrade(migrator):
+    # Remove dispaly name field
+    migrator.drop_column('user', 'display_name')
+    
+    # Add role field
+    migrator.add_column('user', 'role', 'char', max_length=10)
+    
+    # Rename
+    migrator.rename_columen('user', 'password', 'key')
+```
+
+
+### Custom SQL
+
+Peewee Migrate allows running SQL queries directly in the migrations. Here is an example of how to
+do it:
+
+```python
+def upgrade(migrator):
+    migrator.execute_sql('CREATE INDEX idx_user_email ON user (email)')
+
+def downgrade(migrator):
+    migrator.execute_sql('DROP INDEX idx_user_email')
+```
+
+
+### Python script
+
+Peewee Migrate allows running SQL queries directly in the migrations. Here is an example of how to
+do it:
+
+```python
+from package_name.models import models
+
+def upgrade(migrator):
+    migrator.add_column('user', 'display_name', 'char', max_length=58, null=True)
+
+    query = models.User.update(display_name=fn.UPPER(models.User.username))
+    query.execute()
+
+
+def downgrade(migrator):
+    migrator.drop_column('user', 'display_name')
+```
+
+
+## Upgrade from peewee-migrate
+
+Colibris version `<=0.9` used for migrations `peewee-migrate` but starting with version `0.10` it
```

Comment:
I would add some commas and reorganize sentence topic, e.g.:

    Colibris version `<=0.9` used `peewee-migrate` for migrations, but starting with version `0.10`, it uses `peewee-moves`, a more lightweight manager.